app/views/idv/session_errors/throttled.html.erb

```diff
@@ -27,5 +19,21 @@
               ),
             ) %>
       </p>
+      <% if decorated_session.sp_name.present? %>
+        <p>
+          <strong>
+            <%= link_to(
+              t(
+                'idv.cancel.headings.exit.with_sp',
```

Comment:
Though it is currently the same translation, could we introduce a new i18n key to hold this text? Maybe `idv.failure.exit_and_return_to_sp_link`? It looks like we're going to have several screens using this text as a link in the body rather than a heading.